README.md

```diff
@@ -73,8 +73,16 @@ Vircadia consists of many projects and codebases with its unifying structure's g
 
 There are many contributors to Vircadia. Code writers, reviewers, testers, documentation writers, modelers, and general supporters of the project are all integral to its development and success towards its goals. Find out how you can [contribute](CONTRIBUTING.md)!
 
-### [Supporters](https://github.com/sponsors/digisomni/)
+### Support
 
-| [Karol Suprynowicz - 74hc595](https://github.com/ksuprynowicz) | 
+You can support the Vircadia project financially through [GitHub Sponsors](https://github.com/sponsors/digisomni/).
```

Comment:
```suggestion

```
I don't see how sponsoring you is supporting the Vircadia project, but sponsoring David or me isn't.